### ISSUE TITLE
Not being able to save anymore in God of War after adjustment in Cryobyte tool

### ISSUE THREAD

**Issue by gisveld** [CryoByte33/steam-deck-utilities#178]
Hello Kyle,

I installed the Cryobyte tool on my Steam Deck OLED yesterday because I kept experiencing crashes with the game God of War. This game crashes approximately every hour as you also explained in your YouTube video. 

Through the tool, I changed the Swapfilesize to 16 GB and swapiness to value 1 (no further adjustments). After that, I played the game smoothly for 2 hours! So, this was a solution to my problem. 

However, when I later wanted to continue with the game, I noticed that the previous session was not saved. I then replayed 2 hours, but again the game wouldn't save. Even when I try to save manually now, it doesn't work anymore. 

Also, in Steam, it says 32 MB in use for cloud save - 0 bytes available. I don't understand how the settings of Cryobyte can affect the save game folder, but it was a problem immediately after the adjustment. 

Could you please help me with this? I've tried everything, such as temporarily deleting the save games in the Steam folder (so that a new one is created) and then replacing the old saves. Disabling cloud saves also did not work.. The game simply refuses to be saved manually or via autosave. 

Looking forward to hearing from you, Kyle! Greetings from the Netherlands, Hein Bauke

**Comment by CryoByte33** [comment]
Hey there, thanks for using CU!

It sounds like you don't have any more space available on your drive, so you'll need to delete something like another game, reduce the swap file size, or move a game to an SD card. Once you do so you should be able to save your game again!

**Comment by gisveld** [comment]
No, I have plenty of room on my Deck OLED (512 GB). Only 3 games installed. Also, when i start a new game (after backing up my 'real' save file) I can save again. But after replacing the 'real' files back, saving is again not possible... I think my savefile is corrupt. I contact you because it was directly after I activated CU and had a 2.5 hour of (non-freezing) gameplay. So I was very happy that God of War is stable now, but now I can't save any progress anymore. I have also send my savefile to Sante Monica, but do you have some futher advice?

**Comment by CryoByte33** [comment]
Ahhh, I understand, sorry for mistreating the situation. Nothing CU does should affect saving in any way, other than the swap file using the additional space, so I would definitely hazard a guess that there's something up with the file itself in that case. I haven't had that particular issue so I couldn't tell you exactly what the issue is, though.

**Comment by introkun** [comment]
@gisveld How did you replace your 'real' save file back to the original save location? Do you have any *.bak files inside?
